packages/maskbook/src/extension/background-script/EthereumServices/send.ts

```diff
@@ -173,7 +173,9 @@ export async function INTERNAL_send(
                 break
         }
     } catch (error) {
-        callback(error)
+        if (error instanceof Error) {
+            callback(error)
```

Comment:
This change actually changes the semantics and might break the code. Please always re-throw the error